docs/AI-for-security/api/bulk-actions-anonymization-fields-api.asciidoc

```diff
@@ -0,0 +1,233 @@
+
+|==============================================
+
+
+[[bulk-create-anonymization-fields-schema]]
+[discrete]
+==== BulkCreateAction object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`field` |String |A field's name. |Yes
+|`allowed` |Boolean |Defines whether the field will get sent to the LLM. |Yes
+|`anonymized` |Boolean |Defines whether the field will be anonymized if sent to the LLM. Defaults to `false` |Yes
+|==============================================
+
+
+[discrete]
+[[bulk-update-anonymization-fields-schema]]
+===== `BulkUpdateAction` schema
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`allowed` |Boolean |Defines whether the field will get sent to the LLM.  |No
+|`anonymized` |Boolean |Defines whether the field will be anonymized if sent to the LLM. Defaults to `false`  |No
+|==============================================
+
+
+[discrete]
+===== Example requests
+
+*Example 1*
+
+The following request creates 2 anonymization fields:
+
+[source,console]
+--------------------------------------------------
+POST api/security_ai_assistant/anonymization_fields/_bulk_action
+{
+   "create":[
+      {
+         "field":"agent.type",
+         "allowed":true,
+         "anonymized":false
+      },
+      {
+         "field":"agent.name",
+         "allowed":true,
+         "anonymized":true
+      }
+   ],
+}
+
+--------------------------------------------------
+
+[discrete]
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+[discrete]
+===== Response payload
+
+JSON object containing the action's outcome:
+
+- `attributes.summary.total`: Total number of anonymization fields matching the bulk action
+- `attributes.summary.succeeded`: Number of successful outcomes (number of anonymization fields that were created, deleted, or updated)
+- `attributes.summary.failed`: Number of failed outcomes
+- `attributes.summary.skipped`: Number of anonymization fields that were skipped due to various reasons (explained below)
+- `attributes.results.created`: Anonymization field objects that were created during the action's execution
+- `attributes.results.updated`: Anonymization field objects that were updated during the action's execution
+- `attributes.results.deleted`: Anonymization field objects that were deleted during the action's execution
+- `attributes.results.skipped`: Anonymization field that were skipped during the action's execution
+
+An anonymization field can only be `skipped` when the bulk action to be performed on it results in nothing being done. For example, if the `update` action is used to add a field to a prompt that already has that value. Objects returned in `attributes.results.skipped` will only include a prompt's `id`, `name`, and `skip_reason`.
```

Comment:
Copy-paste mix-up :) Please check if the suggestion is correct (I don't think anonymization fields have a `name` but not sure what field it should be?)
```suggestion
An anonymization field can only be `skipped` when the bulk action to be performed on it results in nothing being done. For example, if the `update` action is used to update a field that already has the specified value. Objects returned in `attributes.results.skipped` will only include an anonymization field's `id`, `name`, and `skip_reason`.
```